JavaScript/JavascriptAssignment's.js

```diff
@@ -0,0 +1,317 @@
+// -----------------------------Question - 1--------------------------------
+
+// Create a function that takes two numbers as arguments and returns their sum.
+// addition(3, 2) ➞ 5
+// addition(-3, -6) ➞ -9
+// addition(7, 3) ➞ 10
+
+// ------------------------------Solution----------------------------------
+
+// function addNum(num1,num2){
+//     sum = num1 + num2;
+//     return parseInt(sum);
+// }
+// console.log(`Addition(3 + 2)= ${addNum(3,2)}`)
+// console.log(`Addition(3 + 2)= ${addNum(-3,-6)}`)
+// console.log(`Addition(3 + 2)= ${addNum(7,3)}`)
+
+// -----------------------------Question - 2--------------------------------
+
+// Create a function that takes voltage and current and returns the calculated power
+// circuitPower(230, 10) ➞ 2300
+// circuitPower(110, 3) ➞ 330
+// circuitPower(480, 20) ➞ 9600
+
+// ------------------------------Solution----------------------------------
+
+// function calculatePower(voltage,current){
+//     power = voltage * current;
+//     return parseInt(power);
+// }
+// console.log(`calculatedPower(230 * 10)= ${calculatePower(230, 10)}`)
+// console.log(`calculatedPower(110 * 3)= ${calculatePower(110, 3)}`)
+// console.log(`calculatedPower(480 * 20)= ${calculatePower(480, 20)}`)
+
+// -----------------------------Question - 3--------------------------------
+// Write a function that takes an integer minutes and converts it to seconds.
+// convert(5) ➞ 300
+// convert(3) ➞ 180
+// convert(2) ➞ 120
+// ------------------------------Solution----------------------------------
+
+// function convert(minutes) {
+//     let result = minutes * 60;
+// 	return result;
+// }
+// console.log(`Conversion of Minutes to Seconds(5)=${convert(5)}`)
+// console.log(`Conversion of Minutes to Seconds(3)=${convert(3)}`)
+// console.log(`Conversion of Minutes to Seconds(2)=${convert(2)}`)
+
+// -----------------------------Question - 4--------------------------------
+// Write a function that takes the base and height of a triangle and return its area.
+// triArea(3, 2) ➞ 3
+// triArea(7, 4) ➞ 14
+// triArea(10, 10) ➞ 50
+// ------------------------------Solution----------------------------------
+
+// function areaofTraingle(base,height){
+//     let area = (base * height)/2;
+//     return area;
+// }
+
+// console.log(`Area of Traingle is(3,2)= ${areaofTraingle(3,2)}`)
+// console.log(`Area of Traingle is(7,4)= ${areaofTraingle(7,4)}`)
+// console.log(`Area of Traingle is(10,10)= ${areaofTraingle(10,10)}`)
+
+// -----------------------------Question - 5--------------------------------
+// Create a function that takes the age in years and returns the age in days.
+//  calcAge(65) ➞ 23725
+//  calcAge(0) ➞ 0
+//  calcAge(20) ➞ 7300
+// ------------------------------Solution----------------------------------
+
+// function calcAge(years){
+//     let getAge = years * 365;
```

Comment:
if there is leap year between, can we this condition

Reply:
function calcAge(years){
    let leapDays = (years / 4)
    let getAge = (years * 365) + leapDays;
    return getAge;
}